### PR TITLE
Prep for 3.4.0 release

### PR DESCRIPTION
**Pull request by heitortsergent**
Version bits.

This release adds querying capabilities to the SDK. :dancer:

### CODE REVIEW

File: KeenClient.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
   spec.name         = 'KeenClient'
-  spec.version      = '3.3.1'
+  spec.version      = '3.4'
```
Comment (terryhorner) [inline KeenClient.podspec]: We've used the x.x.x convention for releases so far, so let's tweak this to be 3.4.0 instead of 3.4.

**Comment by heitortsergent** [comment]
@terryhorner updated to 3.4.0 :rocket:

**Comment by terryhorner** [comment]
:+1: :shipit:

**Comment by timfalls** [comment]
Is that it?!

**Comment by heitortsergent** [comment]
@timfalls I haven't pushed the update to CocoaPods. @terryhorner can I go ahead and do it?

**Comment by terryhorner** [comment]
@heitortsergent :ship:

**Comment by heitortsergent** [comment]
@terryhorner done :+1:

**Pull request merged**